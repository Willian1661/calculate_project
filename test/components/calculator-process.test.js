const calculatorProcess = require("../../src/services/calculator-process")
const visorInteractor = require("../../src/services/visor-interactor")

describe("click test", ()=>{


  const resolution = [
  calculatorProcess("1"),
  calculatorProcess("+"),
  calculatorProcess("1"),
  calculatorProcess("="),
]
let visor = ""
let soma = 0
it("'s working click event", ()=>{

for (let i = 0; i < resolution.length; i++) {

visor += resolution[i] + " "

/* if (resolution[i].includes("plus")) {
  let result = Number(resolution[0]) + Number(resolution[2])
  soma += result
}

if (resolution[i].includes("=")) {
} */
    expect(visorInteractor(visor)).toMatchSnapshot()
    
    }  
  })
})