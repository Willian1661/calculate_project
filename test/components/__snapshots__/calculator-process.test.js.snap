// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`click test 's working click event 1`] = `"1 "`;

exports[`click test 's working click event 2`] = `"1 plus "`;

exports[`click test 's working click event 3`] = `"1 plus 1 "`;

exports[`click test 's working click event 4`] = `"1 plus 1 = "`;
